docs/src/getting_started.md

```diff
@@ -211,6 +211,10 @@ reset_rme()  # Reset RME to clear any previous runs
 # Note: if the Julia runtime crashes, check that the specified data file location is correct
 @RME runCreate(name::Cstring, start_year::Cint, end_year::Cint, RCP_scen::Cstring, gcm::Cstring, reps::Cint)::Cint
 
+# Adding dhw tolerance was removed in v1.0.31
+# Add 3 DHW enhancement to outplanted corals
+# set_option("restoration_dhw_tolerance_outplants", 3)
+
```

Comment:
You need to demonstrate what to do with versions > v1.0.31

From the docs and matlab files it looks like they added an `rme_iv_set_outplant_heat_tolerance` function.

The relevant RME functions are:

`ivSetOutplantHeatToleranceMeanDhw` and `ivSetOutplantHeatToleranceSdDhw`

Both of which expect a vector of values for each functional group.

---

Actually, this back and forth has gone for too long. I'm going to accept and merge this as is and address the above later.